### DISCUSSION TITLE
Where does kitty get the powerline symbols from?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#3492] [Q&A] [answered]
Which font does kitty pick up the powerline symbols from?

With the following in my `kitty.conf` file:

```
tab_bar_style powerline
```

I see that kitty correctly displays powerline symbols.  The thing is, I _don't_ have any patched fonts installed.  So I suppose these glyphs must be baked into kitty somehow.

**Comment by kovidgoyal** [comment, marked as answer]
box_drawing.py
